collect_app/src/main/java/org/odk/collect/android/widgets/SpinnerMultiWidget.java

```diff
@@ -117,12 +117,12 @@ public void onClick(DialogInterface dialog, int id) {
 
                                     if (first) {
                                         first = false;
-                                        selectionText.setText(context.getString(R.string.selected)
```

Comment:
Addressed by c1db9e2. I tried to leave as much as possible untouched but it was such a strange way of building the string I had to do some surgery.